.github/workflows/release.yml

```diff
@@ -0,0 +1,36 @@
+name: Release
+on:
+  push:
+    branches:
+      - master
+
+env:
+  GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+
+jobs:
+  build:
+    runs-on: macOS-latest
+
+    steps:
+    - uses: actions/checkout@v2
+
+    - name: extract_version_info
+      run: echo "##[set-output name=version;]$(echo '${{ github.event.head_commit.message }}' | egrep -o '[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}')"
+      id: extract_version_name       
+
+    - uses: release-drafter/release-drafter@v5
+      with:
+          publish: true
+          tag: ${{ steps.extract_version_name.outputs.version }}
+
+    - name: Install Cocoapods
+      run: gem install cocoapods
+      
+    - name: Deploy to Cocoapods
+      run: |
+        set -eo pipefail
+        export LIB_VERSION=${{ steps.extract_version_name.outputs.version }}
```

Comment:
I have one question
Wouldn't it work if declare it on env without manually exporting?

Reply:
It works!
thank you for comment!
06d219c7ef46f9180693f3f9d56cd3eaf2591b6b